README.md

```diff
@@ -38,7 +38,11 @@ Do checkout the Go package which I've written to build Kalam Kernel: [github.com
 * Download the [Latest Version](https://openmtp.ganeshrvel.com/?downloadApp=github&release=stable&platform=mac 'Latest Version')
 
 * Using Homebrew Cask
-  * ```brew install openmtp --cask```
+  * ```brew cask install openmtp```.
+ Note: for older versions of HomeBrew, 
+  ```brew install openmtp --cask```
```

Comment:
shouldnt this be the otherwise?

 ```brew install openmtp --cask``` is for the newer version


How about this?
```suggestion
\* Using Homebrew Cask
\```shell
  # newer versions:
  brew install openmtp --cask

  # older versions:
  brew cask install openmtp
\```
```

^Please remove the escaping character(\)

Reply:
I guess so. I reread their website and it shows
![Screenshot_20210129-071607_Chrome.jpg](https://user-images.githubusercontent.com/23037821/106292834-27ac6c00-6202-11eb-9956-5b0a56d0b4ca.jpg)
But the command shows it the other way in the terminal output. I need to get back to my computer to re-edit

Reply:
Cool, make the edit at your convenience and i will merge it. :)

Reply:
Done